### PR TITLE
Only start required services during gitify

### PR DESCRIPTION
**Pull request by rnathuji**

### CODE REVIEW

**Review by mjoss17 — COMMENTED** [review]
What is this change doing?

**Review by mjoss17 — APPROVED** [review]

**Comment by rnathuji** [comment]
@mjoss17 - For full context :sweat_smile: : Per [this warning](https://docs.google.com/document/d/1yP_BV827GtOP-rscESF78BaVWZ1Bit3DXKB6lubMrQg/edit#bookmark=id.da5ulms4iy52) in our documentation, when you run `gitify` in a `raise-moodle-spikes` environment it will blow away your Moodle database state. On my machine, I actually setup two versions of the repo: 1) `raise-moodle-spikes` and 2) `raise-moodle-spikes-gitify`. I only use the latter when running the `gitify` step so I can keep my "dev" state intact. However, to avoid port collisions I still have to stop services in `raise-moodle-spikes` before running things in `raise-moodle-spikes-gitify`. I found it annoying to have to start all services, so I made this change since all we really need is `moodle`. That way I can do the following:

```bash
$ cd raise-moodle-spikes
$ docker-compose stop moodle
$ cd ../raise-moodle-spikes-gitify
$ ./scripts/gitify_mbz.sh -i input.mbz -o output.mbz
$ docker-compose down
$ cd ../raise-moodle-spikes
$ docker-compose up -d
```

So basically the change in this PR is purely to save some time for things to come down / up.
